Jenkinsfile

```diff
@@ -61,18 +61,51 @@ pipeline {
             }
         }
 
-        stage('Cypress Test') {
+        stage('Fix user rights') {
             steps {
                 catchError(buildResult: 'FAILURE', stageResult: 'FAILURE') {
                     script {
-                        // Fix user rights
                         sh 'sudo chown -R $(id -u):$(id -g) .'
+                    }
+                }
+            }
+        }
+
+        stage('Shared-components Cypress Test') {
+            steps {
+                catchError(buildResult: 'FAILURE', stageResult: 'FAILURE') {
+                    script {
                         sh 'npm run cy:run'
                     }
                 }
             }
         }
 
+        stage('Workbench Cypress Test') {
+            when {
+                expression {
+                    return env.BRANCH_NAME != 'master'
+                }
+            }
+            steps {
+                catchError(buildResult: 'FAILURE', stageResult: 'FAILURE') {
+                    configFileProvider(
+                        [configFile(fileId: 'ceb7e555-a3d9-47c7-9afe-d008fd9efb14', targetLocation: 'graphdb.license')]) {
+                        sh 'cp graphdb.license ./e2e-tests/fixtures/'
+                    }
+                    sh "ls ./e2e-tests/fixtures/"
+                  // --no-ansi suppresses color output that shows as garbage in Jenkins
+                    sh "docker-compose --no-ansi -f docker-compose-test.yaml build --force-rm --no-cache --parallel"
+                    sh "docker-compose --no-ansi -f docker-compose-test.yaml up --abort-on-container-exit --exit-code-from cypress"
+
+                    // Fix coverage permissions
+//                     sh "sudo chown -R \$(id -u):\$(id -g) coverage/"
+//                     sh "sudo chown -R \$(id -u):\$(id -g) cypress/"
+//                     sh "sudo chown -R \$(id -u):\$(id -g) report/"
```

Comment:
Should we still keep these commented out lines?

Reply:
removed

Reply:
removed

Reply:
removed

Reply:
removed

Reply:
removed

Reply:
removed